manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "obsidian-rollover-daily-todos",
   "name": "Rollover Daily Todos",
-  "version": "1.1.3",
+  "version": "1.2.0",

```

Comment:
shouldnt change version in this PR ?

Reply:
Yes, because we add a non-breaking change. So a minor upgrade, according to SEMVER. But: we are currently discussing a better PR with the same functionality by @tim-hilt , so I'll close this one. see https://github.com/lumoe/obsidian-rollover-daily-todos/pull/79 for further discussions.